src/ngx_http_lua_common.h

```diff
@@ -213,6 +217,24 @@ struct ngx_http_lua_main_conf_s {
                      * thus it is safe to store the peer data in the main conf.
                      */
 
+    ngx_chain_t                            *body_filter_chain;
+                    /* body_filter_by_lua does not support yielding and
```

Comment:
Should use `body_filter_by_lua*` instead to cover other variants. Please address other similar places.